subprojects/libcommunist.wrap

```diff
@@ -0,0 +1,7 @@
+[wrap-git]
+url = https://github.com/ProfessorNavigator/libcommunist
+revision = master
+depth = 1
+
+[provide]
+dependency_names = libcommunist
```

Comment:
dependency_names assumes that the subproject uses `meson.override_dependency()`.

In this case you can just use `libcommunist = libcommunist_dep`.

Reply:
Done